cmd/internal/kube/daemonset/kube_events_worker.go

```diff
@@ -306,6 +313,23 @@ func parseBoolConfig(configValue, configName, podName string, defaultValue bool)
 	return parsedValue
 }
 
+// parseSliceConfig parses a slice configuration value, logs errors if parsing fails,
+// and returns the parsed value. Here the configValue should be a JSON string.
+func parseSliceConfig(configValue, configName, podName string) []string {
+	if configValue == "" {
+		return []string{}
+	}
+
+	var parsedValue []string
+	if err := json.Unmarshal([]byte(configValue), &parsedValue); err != nil {
```

Comment:
So the format must be

```
["blah", "foo", "bar"]
```

in this environment variable?  Should we also accept a single string?